tests/testthat/_snaps/slice.md

```diff
@@ -108,17 +110,17 @@
       vec_slice(set_names(letters), c("foo", "bar"))
     Condition
       Error in `vec_slice()`:
-      ! Can't subset elements that don't exist.
-      x Elements `foo` and `bar` don't exist.
+      ! Can't subset elements with `i`.
+      x Can't find elements `foo` and `bar`.
```

Comment:
This is a case where chained errors might fit better because there are multiple bullets "can't / must" bullets describing a problem at different levels.

Same for the "must" cross bullet above.